schema/geometry_schema.json

```diff
@@ -623,6 +635,45 @@
 				"id",
 				"name"
 			]
+		},
+		"PitchedRoofType": {
+			"properties": {
+				"id": {
+					"type": "string"
+				},
+				"name": {
+					"type": [
+						"string",
+						"null"
+					]
+				},
+				"type": {
+					"type": "string",
+					"enum": [
+						"gable",
+						"hip",
+						"shed"
+					]
+				},
+				"pitch": {
+					"description": "Pitch of roof. A value of 6, for example, implies a 6 in 12 pitch (or 26.57 degrees).",
+					"type": "number",
+					"minimum": 0,
+					"exclusiveMinimum": true
+				},
+				"shed_direction": {
+					"description": "Direction of shed roof, from low edge to high edge, in clockwise degrees from the y-axis when viewed from above.",
```

Comment:
"Degrees from y-axis" might be interpreted to be equivalent to "the smaller of degrees-from-north and degrees-from-south" Is it necessary to protect against that?

Reply:
Would stating "the _positive_ y-axis" help? 

I'm open to suggestions :-)

Reply:
"positive y-axis" totally clears it up for me. Thanks!